app/pods/components/agenda/agendaitem/agendaitem-case/agendaitem-titles-edit/component.js

```diff
@@ -37,6 +38,7 @@ export default class SubcaseTitlesEdit extends Component {
       'title': trimText(this.agendaitem.title),
       'shortTitle': trimText(this.agendaitem.shortTitle),
     };
+    propertiesToSetOnSubcase['confidential'] = await this.subcase.get('confidential');
```

Comment:
Why not `propertiesToSetOnSubcase.confidential` ?

Reply:
Nice, de linter zou dat inderdaad ook aangegeven hebben! Goedzo!